src/gemdat/plots/plotly/_density.py

```diff
@@ -210,6 +278,9 @@ def density(volume: Volume,
         else:
             plot_structure(structure, fig=fig)
 
+    if paths:
```

Comment:
I like this option to add the paths directly to `density` plot :+1: 
We should update the notebooks to this standard.

Reply:
Yep, already did it!